### PR TITLE
Blog_Application: Data model

### PR DESCRIPTION
**Pull request by jodavid444**
### In this milestone I implemented the following:

- Make sure that your project has Postgres database set up.
- Build your project schema.
- Create and run the necessary migration files.
- Table and column names should match the ERD diagram.
note: photo for users table should be a link to an image
- Foreign keys should be included.
- Pay attention that in posts table the column holding the foreign key to the users table should be named author_id
- All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+- **Improve styles of the project**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/jodavid444/blog-app/issues)
```
Comment (danmainah) [inline README.md]: - [ ] @jodavid444 Great work on your documentation, however, the URL provided is the wrong one kindly check the project name in the URL it doesn't match this project name.

File: config/database.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem "sqlite3"
+#
+default: &default
+  adapter: sqlite3
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  timeout: 5000
```
Comment (danmainah) [inline config/database.yml]: - [ ] @jodavid444 kindly consider  changing your database from SQLite to Postgresql, this is a project requirement as shown below
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @jodavid444 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ All tables and columns from ERD created and migrations running successfully
✔️ Github flow was correctly used



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (jodavid444) [reply README.md]: Thank you for spending quality time reviewing my project, I have fix the broken link, kindly review it again @danmainah

File: config/database.yml (hunk 2 above)
Comment (jodavid444) [reply config/database.yml]: Thank you for spending quality time reviewing my project, I have updated the database.yml file meeting up to project requirement, kindly review it again @danmainah

**Review by KDlamini — APPROVED** [review]
### Hi @jodavid444 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the project requirements. 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!🎊 🥳 

## Highlights 🌞 

✔️  Good PR title and description
✔️  Database adapter is using Postgresql
✔️  All ERD entities and attributes are created correctly
✔️  Foreign keys and corresponding indexes are created

# Status: Approved ✔️ 

Great-job-smiley

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jodavid444** [comment]
Thanks for the review and approval, keep up the good work @KDlamini

**Pull request merged**